app/models/country_petition_journal.rb

```diff
@@ -40,6 +42,7 @@ def self.reset!
         timezone('utc', now()), timezone('utc', now())
       FROM signatures
       WHERE state = 'validated'
+      AND country <> ''
```

Comment:
Won't this include signatures where country is NULL? Also don't we validate for presence of country so it shouldn't be empty anyway.